packages/frontend/src/features/metricsCatalog/components/MetricTree/MetricTreeFreeGroupNode.tsx

```diff
@@ -0,0 +1,37 @@
+import { Paper, Text, useMantineTheme } from '@mantine/core';
+import type { Node, NodeProps } from '@xyflow/react';
+import { useRef, type FC } from 'react';
+
+export type MetricTreeFreeGroupNodeData = Node;
+
+const MetricTreeFreeGroupNode: FC<NodeProps<MetricTreeFreeGroupNodeData>> = ({
+    height,
+    width,
+}) => {
+    const ref = useRef<HTMLDivElement>(null);
+    const theme = useMantineTheme();
+
+    return (
+        <Paper
+            fz="xs"
+            p="xs"
+            h={height}
+            w={width}
+            bg="gray.0"
+            padding="sm"
+            sx={{
```

Comment:
I thought these sx styles would already be set by default on our MantineTheme
see `MetricsCatalog.tsx`